pkg/client/remotecommand/remotecommand_test.go

```diff
@@ -133,6 +183,9 @@ func (s *fakeUpgradeStream) Close() error {
 }
 
 func (s *fakeUpgradeStream) Reset() error {
+	// send to the channel outside of the lock so we can unblock Read
```

Comment:
It's been a long time since I wrote the original tests and the fake structs to go along with them, but the data race detector kept finding races when I had no locks.